react/src/components/ManageMapProjectPanel/MembersTabContent.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Project } from '@hazmapper/types';
+import { Flex, Table, Card } from 'antd';
+import { useProjectUsers } from '@hazmapper/hooks';
+import { LoadingSpinner, SectionMessage } from '@tacc/core-components';
+
+interface MembersTabProps {
+  project: Project;
+}
+
+const MembersTabContent: React.FC<MembersTabProps> = ({ project }) => {
+  const { data, isLoading, isError, error } = useProjectUsers({
+    projectId: project.id,
+    options: {
+      /* Only fetch if the map is associated with a DS project*/
+      enabled: Boolean(project.system_id.includes('project')),
+    },
+  });
+  if (isLoading) return <LoadingSpinner />;
+  if (isError)
+    return (
+      <SectionMessage type="error">
+        {' '}
+        There was an error loading the members of this project. Error: {error}
+      </SectionMessage>
+    );
+
+  return (
+    <>
+      <Flex vertical justify="center" align="center">
+        {data ? (
+          <Table
+            dataSource={data}
+            rowKey={(data) => data.id}
+            columns={[
+              {
+                title: 'Username',
+                dataIndex: 'username',
+                key: 'username',
+              },
+            ]}
+            pagination={{
+              position: ['bottomCenter'],
+              hideOnSinglePage: true,
+              showSizeChanger: false,
+            }}
+            style={{ width: '90%' }}
+          ></Table>
+        ) : (
+          <Card type="inner">
+            There are no members becuase this map is not saved to a DesignSafe
```

Comment:
what about angular text? I think this was reviewed when this feature was originally implemented.

```
No members as map is not saved to a project. Members are managed by DesignSafe projects.
```